src/main/java/uk/gov/ida/matchingserviceadapter/validators/SubjectValidator.java

```diff
@@ -22,22 +22,18 @@ public void validate(Subject subject, String expectedInResponseTo) {
             throw new SamlResponseValidationException("Subject is missing from the assertion.");
         }
 
-        if (subject.getSubjectConfirmations().size() != 1) {
-            throw new SamlResponseValidationException("Exactly one subject confirmation is expected.");
+        if (subject.getSubjectConfirmations().size() == 0) {
+            throw new SamlResponseValidationException("A subject confirmation is expected.");
         }
 
         SubjectConfirmation subjectConfirmation = subject.getSubjectConfirmations().get(0);
```

Comment:
Do we want to validate the other `SubjectConfirmation` elements if there are more than 1? Also no guarantee the first one is the one we are interested in

Reply:
For this build we are just removing any checks that are not present in the old MSA. We are going to play a play a ticket to add back additional checks when we have time to test against example SAML from the IDPs.